examples/migrating_trench/trench_example.py

```diff
@@ -0,0 +1,168 @@
+"""
+Migrating Trench Test case
+=======================
+
+Solves the test case of a migrating trench.
+
+We use this test case to validate the implementation of the mathematical and
+numerical methods used in Thetis to model sediment transport and morphological changes.
+In the figure produced, we compare our results with experimental data from a lab study
+
+For more details, see
+[1] Clare et al. 2020. “Hydro-morphodynamics 2D Modelling Using a Discontinuous
+    Galerkin Discretisation.” EarthArXiv. January 9. doi:10.31223/osf.io/tpqvy.
+
+"""
+
+from thetis import *
+
+import numpy as np
+import pandas as pd
+import pylab as plt
+import time
+
+conservative = False
+
+# Note it is necessary to run trench_hydro first to get the hydrodynamics simulation
+
+# define mesh
+lx = 16
+ly = 1.1
+nx = lx*5
+ny = 5
+mesh2d = RectangleMesh(nx, ny, lx, ly)
+
+x, y = SpatialCoordinate(mesh2d)
+
+# define function spaces
+V = FunctionSpace(mesh2d, "CG", 1)
+DG_2d = FunctionSpace(mesh2d, "DG", 1)
+vector_dg = VectorFunctionSpace(mesh2d, "DG", 1)
+
+# define underlying bathymetry
+bathymetry_2d = Function(V, name='bathymetry_2d')
+initialdepth = Constant(0.397)
+depth_riv = Constant(initialdepth - 0.397)
+depth_trench = Constant(depth_riv - 0.15)
+depth_diff = depth_trench - depth_riv
+
+trench = conditional(le(x, 5), depth_riv, conditional(le(x, 6.5), (1/1.5)*depth_diff*(x-6.5) + depth_trench,
+                     conditional(le(x, 9.5), depth_trench, conditional(le(x, 11), -(1/1.5)*depth_diff*(x-11) + depth_riv, depth_riv))))
+bathymetry_2d.interpolate(-trench)
+
+# choose directory to output results
+ts = time.time()
+st = datetime.datetime.fromtimestamp(ts).strftime('%Y-%m-%d %H:%M:%S')
+outputdir = 'outputs' + st
```

Comment:
is it necessary to add the date stamp in the output file in this example? we do not use it elsewhere

Reply:
I have removed this